src/azul/service/hca_response_v5.py

```diff
@@ -74,7 +74,14 @@ class PaginationObj(JsonObject):
 
 
 class FileTypeSummary(JsonObject):
+    # FIXME: Remove deprecated field 'fileType'
+    #        https://github.com/DataBiosphere/azul/issues/3180
     fileType = StringProperty()
+    format = StringProperty()
+    # FIXME: Remove deprecated field 'source'
+    #        https://github.com/DataBiosphere/azul/issues/3180
+    source = ListProperty()  # List could have string(s) and/or None
+    fileSource = ListProperty()  # List could have string(s) and/or None
```

Comment:
```suggestion
    fileSource = ListProperty()  # List could have string(s) and/or None
    # FIXME: Remove deprecated field 'source'
    #        https://github.com/DataBiosphere/azul/issues/3180
    source = fileSource
```
normal code
fixme
backwards compatibility code